lib/src/drawings.dart

```diff
@@ -2,27 +2,21 @@ import "dart:convert";
 import "dart:typed_data";
 
 import "package:flutter/material.dart";
+import "package:flutter_gyw/flutter_gyw.dart";
```

Comment:
Inside src folder, I prefer to call the src file you need instead of the entire library to prevent cycle import.